lib/foreman_tasks/dynflow/persistence.rb

```diff
@@ -11,7 +11,8 @@ def save_execution_plan(execution_plan_id, value)
         begin
           on_execution_plan_save(execution_plan_id, value)
```

Comment:
https://github.com/theforeman/foreman-tasks/pull/235/files#diff-a00196c0418a614248a22c431ef0d163R22 - after changing some stuff in the helper, to support `def humanize_name` etc.. methods in the job, I got it to show all the information just fine